src/main/java/de/tum/in/www1/artemis/service/QuizExerciseImportService.java

```diff
@@ -149,6 +154,36 @@ else if (quizQuestion instanceof ShortAnswerQuestion saQuestion) {
         newExercise.setQuizQuestions(importedExercise.getQuizQuestions());
     }
 
+    /**
+     * Checks whether an alleged backgroundFilePath is actually one. In case it isn't, a "Background File Path is not valid!"-IllegalArgumentException is thrown.
+     *
+     * @param backgroundFilePath Path to be checked
+     */
+    public static void checkIfBackgroundFilePathIsABackgroundFilePathElseThrow(String backgroundFilePath) {
+        /*
+         * FileService.copyExistingFileToTarget(String, String, Long) only acts,
+         * when backgroundFilePath != null && !backgroundFilePath.contains("files/temp")
+         * => No need to check if backgroundFilePath == null || backgroundFilePath.contains("files/temp")
+         */
+        if (backgroundFilePath != null && !backgroundFilePath.contains("files/temp")) {
+            // Substitutes all escape sequences in the backgroundFilePath with their standard elements
+            String unescapedBackgroundFilePath = StringEscapeUtils.unescapeJava(backgroundFilePath);
+            // Removes redundant elements (e.g. ../ or ./) from the backgroundFilePath
+            Path normalisedBackgroundFilePath = Paths.get(unescapedBackgroundFilePath).normalize();
+            // Resolve the backgroundFilePath on the empty path in order to receive an absolute path
+            Path absoluteBackgroundFilePath = Path.of("").resolve(normalisedBackgroundFilePath);
```

Comment:
I will change this in my PR, so it would make sense if we could already adapt this here:
These are no file paths but partial URIs. So please work with `URI` instead.

Reply:
Introduced `URI.create(dndQuestion.getBackgroundFilePath())` in [69a0c8b408b16a8e68ace79dfa4cee7927e48db0](https://github.com/ls1intum/Artemis/pull/7170/commits/69a0c8b408b16a8e68ace79dfa4cee7927e48db0).